packages/aws-rfdk/lib/core/lambdas/nodejs/version-provider/handler.ts

```diff
@@ -0,0 +1,357 @@
+/**
```

Comment:
This entire file has to do directly with Deadline so why is it in the Core library of the rfdk instead of the Deadline section.